_haikus/The_Presence.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: The Presence
+author: Hanike
+---
+
+There is a presence:<br>
+Silently, the cat watches you<br>
```

Comment:
This line needs 1 fewer syllable

Reply:
You can probably remove "you" here to be:
`Silently, the cat watches<br>`

Reply:
Same as the other.. it really seems Hairoku has some counting issues.

Reply:
![hairoku-presence-count](https://user-images.githubusercontent.com/1890948/47900478-d616aa80-de5b-11e8-80b6-7e7941689225.PNG)